package.yaml

```diff
@@ -1,5 +1,5 @@
 name:        typed-process
-version:     0.2.5.0
+version:     0.2.5.1
```

Comment:
Version bump should be to 0.2.6.0 instead. It's arguably a major change (0.3.0.0), but I'd rather stick with 0.2.6.0 for this.